docs/index.md

```diff
@@ -1,27 +1,279 @@
-title: 'Quick start'
+sidebar_label: 'Getting started'
+title: 'Getting started'
 ---
 
 # Apify API client for JavaScript
 
-`apify-client` is the official library to access [Apify API](https://docs.apify.com/api/v2) from your
-JavaScript applications. It runs both in Node.js and browser and provides useful features like
-automatic retries and convenience functions that improve the experience of using the Apify API.
+`apify-client` is the official library to access the [Apify REST API](https://docs.apify.com/api/v2) from your
+JavaScript/TypeScript applications. It runs both in Node.js and browser and provides useful features like
+automatic retries and convenience functions that improve the experience of using the Apify API. All requests and responses (including errors) are encoded in JSON format with UTF-8 encoding.
 
-You can install the client via the [npm package](https://www.npmjs.com/package/apify-client). To do that, simply run `npm i apify-client`.
+## Pre-requisites
 
-## Quick Start
+`apify-client` requires Node.js version 16 or higher. Node.js is available for download on the [official website](https://nodejs.org/). Check for your current node version by running:
+
+```bash
+node -v
+```
+
+## Installation
+
+You can install the client via [NPM](https://www.npmjs.com/) or use any other package manager of your choice.
+
+```bash
+npm i apify-client
+# or
+pnpm add apify-client
+# or
+yarn add apify-client
+# or
+bun add apify-client
+```
+
+## Authentication and Initialization
+
+To use the client, you need an [API token](https://docs.apify.com/platform/integrations/api#api-token). You can find your token under [Integrations](https://console.apify.com/account/integrations) tab in Apify Console. Copy the token and initialize the client by providing the token (`MY-APIFY-TOKEN`) as a parameter to the `ApifyClient` constructor.
 
 ```js
+// ES5 example import
 const { ApifyClient } = require('apify-client');
+```
+
+```js
+// ES6+ example import
+import { ApifyClient } from 'apify';
+```
 
+```js
+// Client initialization with the API token
 const client = new ApifyClient({
     token: 'MY-APIFY-TOKEN',
 });
+```
+
+
+:::warning Secure access
 
-// Starts an actor and waits for it to finish.
-const { defaultDatasetId } = await client.actor('john-doe/my-cool-actor').call();
-// Fetches results from the actor's dataset.
+The API token is used to authorize your requests to the Apify API. You can be charged for the usage of the underlying services, so do not share your API token with untrusted parties or expose it on the client side of your applications
+
+:::
+
+
+## Quick start
+
+One of the most common use cases is starting [Actors](https://docs.apify.com/platform/actors) (serverless programs running in the [Apify cloud](https://docs.apify.com/platform)) and getting results from their [datasets](https://docs.apify.com/platform/storage/dataset) (storage) after they finish the job (usually scraping, automation processes or data processing).
```

Comment:
nit: links to the docs don't have to be absolute, e.g. `/platform/actors` will also work

Reply:
Fixed all the absolute links